stdlib/Mmap/src/Mmap.jl

```diff
@@ -45,6 +45,7 @@ Anonymous() = Anonymous("",false,true)
 Base.isopen(::Anonymous) = true
 Base.isreadable(::Anonymous) = true
 Base.iswritable(a::Anonymous) = !a.readonly
+Base.isfile(::Anonymous) = true
```

Comment:
Seems counter to the definition of Anonymous but I believe it's what it needs to be to pass the logic we're adding.
Perhaps there needs to be a `isresizable` or something in the API?

Reply:
```suggestion
```